backend/testForm.html

```diff
@@ -0,0 +1,18 @@
+<div class="form-box">
+  <h1>Simple Contact Form</h1>
+  <form action="http://localhost:8000/tournaments" method="post">
```

Comment:
Is this testForm.html still needed? I would expect the form in your frontend to be the one used for testing.

Reply:
Deleted